charts/prometheus-redis-exporter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: v1.54.0
+appVersion: v1.55.0
```

Comment:
```suggestion
appVersion: v1.54.1
```

Reply:
why? there is no 1.54.1 anyway. https://github.com/oliver006/redis_exporter/releases